### ISSUE TITLE
fails to connect and have these 2 screens

### ISSUE THREAD

**Issue by Exiledslinky** [rowengb/GSPro-MLM2PRO-OCR#10]
these are the 2 error screens ive had so far

https://imagetolink.com/ib/YIbW1PkC5S
https://imagetolink.com/ib/v2dKFBBqKK

**Comment by wonder99** [comment]
The message mentioning 'invalid window handle' normally arises because the WINDOW_NAME in your settings.json doesn't match the actual window name.  Which screen mirroring app are you using?
The 'no connection could be made' normally arises because the GSPConnect.exe program is not running. Can you confirm it is?  That program may need to be launched fresh each time before this connector is launched.

**Comment by Exiledslinky** [comment]
Yeah figures out what I was doing wrong works great now.   Just need putting enabled like the r10
